src/core/a-entity.js

```diff
@@ -266,10 +266,6 @@ var proto = Object.create(ANode.prototype, {
         // Add component if it is a dependency and not yet defined.
         this.setAttribute(name, '');
       } else {
-        if (this.isScene && !this.hasAttribute(name) && name in this.defaultComponents) {
-          // For scene default components, expose them in the DOM.
-          HTMLElement.prototype.setAttribute.call(this, name, this.defaultComponents[name]);
```

Comment:
yeah, I follow. it'd be hacky, but we could like ignore reinitialising those components twice upon those attribute updates for those particular components on the scene. it'd be a mess but yeah.